terraform/modules/hub/hub_policy.tf

```diff
@@ -56,12 +76,24 @@ locals {
   LOCATIONS
 
   nginx_policy_location_blocks_base64 = base64encode(local.policy_location_blocks)
+
+  policy_fargate_location_blocks = <<-LOCATIONS
+  location = /prometheus/metrics {
+    proxy_pass http://localhost:8081;
+    proxy_set_header Host policy.${local.root_domain};
+  }
+  location / {
+    proxy_pass http://localhost:8080;
+    proxy_set_header Host policy.${local.root_domain};
+  }
+  LOCATIONS
 }
 
 data "template_file" "policy_task_def" {
   template = file("${path.module}/files/tasks/hub-policy.json")
 
   vars = {
+    optional_links                = "\"links\": [\"policy\"],"
```

Comment:
**non-blocking**: this looks unfortunately fragile ... would you consider setting `optional_links = ["policy"]` ... then using `"links": ${json_encode(optional_links)},` in the template? 

...or is there an issue when `links` is present even if empty list in some scenarios :shrug:

Reply:
`optional_links` will get deleted when we delete old policy-ecs-ec2